shoop/configuration.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+API for shoop dynamic configurations. Values is stored to database in
+`shoop.core.configurations.ConfigurationItem model`. Shop specific
+configurations is cached.
+
+Shop can be None which means that configuration item is global. For shop
+first all global configurations is read and then shop specific
+configuration which may override global configurations.
+"""
```

Comment:
* Module docstring should have a title too.
* It's "dynamic configuration", not "dynamic configurations".
* "Values is" -> "Values are"
* Drop the "model" word from the quotes or Sphinx won't find the class: It should be ``` `ConfigurationItem` model ``` rather than ``` `ConfigurationItem model` ```)

Maybe just change the first sentence to title:
```python
API for Shoop's Dynamic Configuration.

Values are stored to database in `ConfigurationItem` model. ...
```